packages/enhanced_gradients/README.md

```diff
@@ -0,0 +1,46 @@
+# enhanced_gradients
+
+This package adds an easy way to enhance your Flutter gradients by interpolating colors
+in the HCT color system provided by the [material_color_utilities package](https://pub.dev/packages/material_color_utilities).
+
+## Usage
+
+The package exposes two ways to modify the builtin `LinearGradient`, `RadialGradient` and `SweepGradient` gradients:
+
+```dart
+// 1st way: extension method
+
+LinearGradient(
+  colors: const [Color(0xFF000000), Color(0xFFFFFFFF)],
+  // ...
+).enhanced()
+
+RadialGradient(
+  colors: const [Color(0xFF000000), Color(0xFFFFFFFF)],
+  // ...
+).enhanced()
+
+SweepGradient(
+  colors: const [Color(0xFF000000), Color(0xFFFFFFFF)],
+  // ...
+).enhanced()
+```
+
+```dart
+// 2nd way: `Enhanced*Gradient` class
+
+EnhancedLinearGradient(
+  colors: const [Color(0xFF000000), Color(0xFFFFFFFF)],
+  // ...
+)
+
+EnhancedRadialGradient(
+  colors: const [Color(0xFF000000), Color(0xFFFFFFFF)],
+  // ...
+).enhanced()
```

Comment:
Those extension methods shouldn't be here I suppose

Reply:
Whoopsie 😅